recap/src/lib.rs

```diff
@@ -135,9 +163,9 @@ type Result<T> = envy::Result<T>;
 
 struct Vars<'a, Iter>(Iter)
 where
-    Iter: IntoIterator<Item = (&'a str, &'a str)>;
+    Iter: IntoIterator<Item = (&'a str, &'a str, Option<&'a FieldOptions>)>;
```

Comment:
I have a follow-up PR that changes all these duplicated `(&'a str, &'a str, Option<&'a FieldOptions>)` into `Val<'a>` but figured that might be better as a separate change. Happy to include it here if you'd rather though!